src/runtime/virtcontainers/clh.go

```diff
@@ -73,7 +73,12 @@ const (
 	// Values are mandatory by http API
 	// Values based on:
 	clhTimeout    = 10
-	clhAPITimeout = 1
+
+	// Pods which request large memory fail to boot within the bootvm
+	// timeout when using memory config shared=true, ex. using virtio_fs
+	// for filesystem share.
+	// This larger timeout allows for pods to boot successfully.
+	clhAPITimeout = 100
```

Comment:
@danmihai1 do we want to change

```
	// TODO: review this 10 second minimum timeout value.
	if bootvm_timeout < 10 {
		bootvm_timeout = 10
	}
``` in `StartVm`?

Reply:
I see the following change upstream: https://github.com/kata-containers/kata-containers/commit/f4509b806ba9f930129194a0e08bb439a7ed392c

Maybe cherry-pick the upstream change here as part of this PR or just leave the PR as is and close my comment

Reply:
Indeed, I would use clhCreateAndBootVMMinimumTimeout here. And, that should be a separate commit + PR - since it's not directly related to the clh.vmconfig.Memory.Shared change.

Reply:
Sure Thanks. I will make those changes in a separate PR.